### DISCUSSION TITLE
Change default landing page to public timeline

### DISCUSSION THREAD

**Discussion by newbold** [mastodon/mastodon#18815] [General]
Is there any way to set my instance’s default landing page to its public timeline? Thanks!

**Comment by DavidGerhart** [comment]
It's been a year since this post.  And this is a great idea. So I'm surprised that there isn't an Administration item that at least permits assigning the default public landing page.

**Comment by russtanner** [comment]
Agreed. Having a default/configurable landing page that displays active content would help retain visitors. I would love to see the default landing page be configurable.
